### ISSUE TITLE
1.16-alpine not updated to nginx-1.16.1-r2

### ISSUE THREAD

**Issue by pedantic79** [nginxinc/docker-nginx#388]
The latest 1.16-alpine container still shows 1.16.1-r1:

```
/ # apk list | grep nginx
nginx-1.16.1-r1 x86_64 {nginx} (2-clause BSD-like license) [installed]
nginx-module-xslt-1.16.1-r1 x86_64 {nginx-module-xslt} (2-clause BSD-like license) [installed]
nginx-module-image-filter-1.16.1-r1 x86_64 {nginx-module-image-filter} (2-clause BSD-like license) [installed]
nginx-module-njs-1.16.1.0.3.8-r1 x86_64 {nginx-module-njs} (2-clause BSD-like license) [installed]
nginx-module-geoip-1.16.1-r1 x86_64 {nginx-module-geoip} (2-clause BSD-like license) [installed]
```

However the [latest version](https://pkgs.alpinelinux.org/packages?name=nginx&branch=v3.10) for alpine-3.10 is 1.16.1-r2. The security scanner I'm using is reporting that the ngnix:1.16-alpine is vulnerable to CVE-2019-20372.

**Comment by thresheek** [comment]
Hi @pedantic79.

Hope all is well.

Our images do not use alpinelinux.org repositories but rather our own nginx.org builds, which, in turn, are vulnerable to the CVE in question.

For the time being, I can only advise using mainline/1.17 tags if you feel you need to have that CVE mitigated.  I'll investigate on what could be done to fix that CVE in 1.16 images as well.

Thank you!

**Comment by thresheek** [comment]
Since there was no upstream release for an upstream 1.16 branch it will not be fixed.  Please use 1.18, which is the new stable.
